packages/headless/src/api/commerce/product-listings/product-listing-params.ts

```diff
@@ -17,13 +49,17 @@ export const baseProductListingRequest = (
   method: HttpMethods,
   contentType: HTTPContentType,
   queryStringArguments: Record<string, string> = {}
-) => {
-  const baseUrl = `${req.url}/rest/organizations/${req.organizationId}/commerce/products/listing`;
+): Pick<
+  PlatformClientCallOptions,
+  'accessToken' | 'method' | 'contentType' | 'url'
+> => {
+  const {baseClientUrl, organizationId, accessToken} = req;
```

Comment:
I'm wondering if `baseClientUrl` is needed on top of `platformUrl`? It's easier to add parameters than remove/maintain them.